client/web/src/user/settings/accessTokens/UserSettingsCreateAccessTokenCallbackPage.tsx

```diff
@@ -173,12 +198,15 @@ export const UserSettingsCreateAccessTokenCallbackPage: React.FC<Props> = ({
                     switchMap(() =>
                         (requester ? createAccessToken(user.id, [AccessTokenScopes.UserAll], note) : NEVER).pipe(
                             tap(result => {
-                                // SECURITY: If the request was from a valid requestor, redirect to the allowlisted redirect URL.
+                                // SECURITY: If the request was from a valid requester, redirect to the allowlisted redirect URL.
                                 // SECURITY: Local context ONLY
                                 if (requester) {
                                     onDidCreateAccessToken(result)
                                     setNewToken(result.token)
-                                    const uri = replaceToken(requester?.redirectURL, result.token)
+                                    let uri = replaceToken(requester?.redirectURL, result.token)
+                                    if (requestFrom === 'JETBRAINS') {
```

Comment:
Nit: to avoid `port ? '11111'`. We should avoid redirecting to a hardcoded number.

```suggestion
                                    if (requestFrom === 'JETBRAINS' && port) {
```

Reply:
Added this, thanks!